Web-nodejs fails to build

The ui container image fails to build.
Because of this the coolstore GUI is unavailable and takes away from the experience.

On RHPDS, when starting a build for the ui:
"
An error occurred while starting the build.
 Reason: Error resolving ImageStreamTag nodejs:4 in namespace openshift: unable to find latest tagged image
"

Setting the builder to nodejs:6 :
oc logs web-ui-1-build -n openshift
Using docker-registry.default.svc:5000/openshift/nodejs@sha256:9a9b1e60650ef4060957ef8fc3a89f63a996d794c7be55c5f233671cace8945f as the s2i builder image
---> Installing application source ...
---> Installing all dependencies
npm WARN deprecated cryptiles@2.0.5: This version is no longer maintained. Please upgrade to the latest version.
npm WARN deprecated boom@2.10.1: This version is no longer maintained. Please upgrade to the latest version.
npm WARN deprecated hoek@2.16.3: This version is no longer maintained. Please upgrade to the latest version.

> web@1.0.0 postinstall /opt/app-root/src
> bower --config.registry.search=${BOWER_MIRROR} install

bower imagesloaded#4.1.1    not-cached https://github.com/desandro/imagesloaded.git#4.1.1
bower imagesloaded#4.1.1       resolve https://github.com/desandro/imagesloaded.git#4.1.1
bower patternfly#3.17.0     not-cached https://github.com/patternfly/patternfly.git#3.17.0
bower patternfly#3.17.0        resolve https://github.com/patternfly/patternfly.git#3.17.0
bower angular-patternfly#3.17.0       not-cached https://github.com/patternfly/angular-patternfly.git#3.17.0
bower angular-patternfly#3.17.0          resolve https://github.com/patternfly/angular-patternfly.git#3.17.0
bower isotope#2.2.2                   not-cached https://github.com/metafizzy/isotope.git#2.2.2
bower isotope#2.2.2                      resolve https://github.com/metafizzy/isotope.git#2.2.2
bower angular-route#1.5.10            not-cached https://github.com/angular/bower-angular-route.git#1.5.10
bower angular-route#1.5.10               resolve https://github.com/angular/bower-angular-route.git#1.5.10
bower keycloak#1.9.8                  not-cached https://github.com/keycloak/keycloak-js-bower.git#1.9.8
bower keycloak#1.9.8                     resolve https://github.com/keycloak/keycloak-js-bower.git#1.9.8
bower bootstrap-switch#3.3.2          not-cached https://github.com/nostalgiaz/bootstrap-switch.git#3.3.2
bower bootstrap-switch#3.3.2             resolve https://github.com/nostalgiaz/bootstrap-switch.git#3.3.2
bower isotope#2.2.2                     download https://github.com/metafizzy/isotope/archive/v2.2.2.tar.gz
bower patternfly#3.17.0                 download https://github.com/patternfly/patternfly/archive/v3.17.0.tar.gz
bower imagesloaded#4.1.1                download https://github.com/desandro/imagesloaded/archive/v4.1.1.tar.gz
bower angular-patternfly#3.17.0         download https://github.com/patternfly/angular-patternfly/archive/v3.17.0.tar.gz
bower angular-route#1.5.10              download https://github.com/angular/bower-angular-route/archive/v1.5.10.tar.gz
bower bootstrap-switch#3.3.2            download https://github.com/nostalgiaz/bootstrap-switch/archive/v3.3.2.tar.gz
bower keycloak#1.9.8                    download https://github.com/keycloak/keycloak-js-bower/archive/1.9.8.tar.gz
bower isotope#2.2.2                      extract archive.tar.gz
bower isotope#2.2.2                     resolved https://github.com/metafizzy/isotope.git#2.2.2
bower get-size#~1.2.2                 not-cached https://github.com/desandro/get-size.git#~1.2.2
bower get-size#~1.2.2                    resolve https://github.com/desandro/get-size.git#~1.2.2
bower masonry#~3.3.0                  not-cached https://github.com/desandro/masonry.git#~3.3.0
bower masonry#~3.3.0                     resolve https://github.com/desandro/masonry.git#~3.3.0
bower fizzy-ui-utils#~1.0.1           not-cached https://github.com/metafizzy/fizzy-ui-utils.git#~1.0.1
bower fizzy-ui-utils#~1.0.1              resolve https://github.com/metafizzy/fizzy-ui-utils.git#~1.0.1
bower keycloak#1.9.8                     extract archive.tar.gz
bower angular-route#1.5.10               extract archive.tar.gz
bower imagesloaded#4.1.1                 extract archive.tar.gz
bower keycloak#1.9.8                    resolved https://github.com/keycloak/keycloak-js-bower.git#1.9.8
bower matches-selector#~1.0.2         not-cached https://github.com/desandro/matches-selector.git#~1.0.2
bower matches-selector#~1.0.2            resolve https://github.com/desandro/matches-selector.git#~1.0.2
bower angular-route#1.5.10              resolved https://github.com/angular/bower-angular-route.git#1.5.10
bower angular#1.5.10                  not-cached https://github.com/angular/bower-angular.git#1.5.10
bower angular#1.5.10                     resolve https://github.com/angular/bower-angular.git#1.5.10
bower imagesloaded#4.1.1                resolved https://github.com/desandro/imagesloaded.git#4.1.1
bower patternfly#3.17.0                  extract archive.tar.gz
bower angular-patternfly#3.17.0          extract archive.tar.gz
bower bootstrap-switch#3.3.2             extract archive.tar.gz
bower get-size#~1.2.2                   download https://github.com/desandro/get-size/archive/v1.2.2.tar.gz
bower outlayer#~1.4.1                 not-cached https://github.com/metafizzy/outlayer.git#~1.4.1
bower outlayer#~1.4.1                    resolve https://github.com/metafizzy/outlayer.git#~1.4.1
bower fizzy-ui-utils#~1.0.1             download https://github.com/metafizzy/fizzy-ui-utils/archive/v1.0.1.tar.gz
bower masonry#~3.3.0                    download https://github.com/desandro/masonry/archive/v3.3.2.tar.gz
bower matches-selector#~1.0.2           download https://github.com/desandro/matches-selector/archive/v1.0.3.tar.gz
bower bootstrap-switch#3.3.2            resolved https://github.com/nostalgiaz/bootstrap-switch.git#3.3.2
bower bootstrap#>=2.3.2               not-cached https://github.com/twbs/bootstrap.git#>=2.3.2
bower bootstrap#>=2.3.2                  resolve https://github.com/twbs/bootstrap.git#>=2.3.2
bower jquery#>=1.9.0                  not-cached https://github.com/jquery/jquery-dist.git#>=1.9.0
bower jquery#>=1.9.0                     resolve https://github.com/jquery/jquery-dist.git#>=1.9.0
bower angular#1.5.10                    download https://github.com/angular/bower-angular/archive/v1.5.10.tar.gz
bower jquery#>=1.9.0                    download https://github.com/jquery/jquery-dist/archive/3.3.1.tar.gz
bower outlayer#~1.4.1                   download https://github.com/metafizzy/outlayer/archive/v1.4.2.tar.gz
bower get-size#~1.2.2                    extract archive.tar.gz
bower get-size#~1.2.2                   resolved https://github.com/desandro/get-size.git#1.2.2
bower get-style-property#1.x          not-cached https://github.com/desandro/get-style-property.git#1.x
bower get-style-property#1.x             resolve https://github.com/desandro/get-style-property.git#1.x
bower matches-selector#~1.0.2            extract archive.tar.gz
bower fizzy-ui-utils#~1.0.1              extract archive.tar.gz
bower matches-selector#~1.0.2           resolved https://github.com/desandro/matches-selector.git#1.0.3
bower fizzy-ui-utils#~1.0.1             resolved https://github.com/metafizzy/fizzy-ui-utils.git#1.0.1
bower doc-ready#~1.0.4                not-cached https://github.com/desandro/doc-ready.git#~1.0.4
bower doc-ready#~1.0.4                   resolve https://github.com/desandro/doc-ready.git#~1.0.4
bower masonry#~3.3.0                     extract archive.tar.gz
bower bootstrap#>=2.3.2                 download https://github.com/twbs/bootstrap/archive/v4.2.1.tar.gz
bower patternfly#3.17.0             invalid-meta for:/tmp/32faccfb063ec5f39cbb248071d6f2c7/bower/ecd5a67191fbb1ae56f51993f1a892a3-23-uGy476/bower.json
bower patternfly#3.17.0             invalid-meta The "main" field has to contain only 1 file per filetype; found multiple .css files: ["dist/css/patternfly.css","dist/css/patternfly-additions.css"]
bower masonry#~3.3.0                    resolved https://github.com/desandro/masonry.git#3.3.2
bower angular-patternfly#3.17.0         resolved https://github.com/patternfly/angular-patternfly.git#3.17.0
bower angular-animate#1.3.0 - 1.5.*   not-cached https://github.com/angular/bower-angular-animate.git#1.3.0 - 1.5.*
bower angular-animate#1.3.0 - 1.5.*      resolve https://github.com/angular/bower-angular-animate.git#1.3.0 - 1.5.*
bower angular-sanitize#1.3.0 - 1.5.*  not-cached https://github.com/angular/bower-angular-sanitize.git#1.3.0 - 1.5.*
bower angular-sanitize#1.3.0 - 1.5.*     resolve https://github.com/angular/bower-angular-sanitize.git#1.3.0 - 1.5.*
bower lodash#3.x                      not-cached https://github.com/lodash/lodash.git#3.x
bower lodash#3.x                         resolve https://github.com/lodash/lodash.git#3.x
bower outlayer#~1.4.1                    extract archive.tar.gz
bower get-style-property#1.x            download https://github.com/desandro/get-style-property/archive/v1.0.4.tar.gz
bower outlayer#~1.4.1                   resolved https://github.com/metafizzy/outlayer.git#1.4.2
bower get-style-property#~1.0.4       not-cached https://github.com/desandro/get-style-property.git#~1.0.4
bower get-style-property#~1.0.4          resolve https://github.com/desandro/get-style-property.git#~1.0.4
bower doc-ready#~1.0.4                  download https://github.com/desandro/doc-ready/archive/v1.0.4.tar.gz
bower get-style-property#~1.0.4         download https://github.com/desandro/get-style-property/archive/v1.0.4.tar.gz
bower eventie#~1.0.3                  not-cached https://github.com/desandro/eventie.git#~1.0.3
bower eventie#~1.0.3                     resolve https://github.com/desandro/eventie.git#~1.0.3
bower eventEmitter#>=4.2 <5           not-cached https://github.com/Olical/EventEmitter.git#>=4.2 <5
bower eventEmitter#>=4.2 <5              resolve https://github.com/Olical/EventEmitter.git#>=4.2 <5
bower jquery#>=1.9.0                     extract archive.tar.gz
bower angular-bootstrap#0.14.x        not-cached https://github.com/angular-ui/bootstrap-bower.git#0.14.x
bower angular-bootstrap#0.14.x           resolve https://github.com/angular-ui/bootstrap-bower.git#0.14.x
bower patternfly#3.17.0                 resolved https://github.com/patternfly/patternfly.git#3.17.0
bower bootstrap#~3.3.7                not-cached https://github.com/twbs/bootstrap.git#~3.3.7
bower bootstrap#~3.3.7                   resolve https://github.com/twbs/bootstrap.git#~3.3.7
bower jquery#~2.1.4                   not-cached https://github.com/jquery/jquery-dist.git#~2.1.4
bower jquery#~2.1.4                      resolve https://github.com/jquery/jquery-dist.git#~2.1.4
bower angular#1.5.10                     extract archive.tar.gz
bower bootstrap#~3.3.7                  download https://github.com/twbs/bootstrap/archive/v3.3.7.tar.gz
bower jquery#~2.1.4                     download https://github.com/jquery/jquery-dist/archive/2.1.4.tar.gz
bower datatables#~1.10.12             not-cached https://github.com/DataTables/DataTables.git#~1.10.12
bower datatables#~1.10.12                resolve https://github.com/DataTables/DataTables.git#~1.10.12
bower bootstrap-datepicker#~1.6.4     not-cached https://github.com/eternicode/bootstrap-datepicker.git#~1.6.4
bower bootstrap-datepicker#~1.6.4        resolve https://github.com/eternicode/bootstrap-datepicker.git#~1.6.4
bower datatables-colvis#~1.1.2        not-cached https://github.com/DataTables/ColVis.git#~1.1.2
bower datatables-colvis#~1.1.2           resolve https://github.com/DataTables/ColVis.git#~1.1.2
bower c3#~0.4.11                      not-cached https://github.com/masayuki0812/c3.git#~0.4.11
bower c3#~0.4.11                         resolve https://github.com/masayuki0812/c3.git#~0.4.11
bower font-awesome#~4.6.3             not-cached https://github.com/FortAwesome/Font-Awesome.git#~4.6.3
bower font-awesome#~4.6.3                resolve https://github.com/FortAwesome/Font-Awesome.git#~4.6.3
bower matchHeight#~0.7.0              not-cached https://github.com/liabru/jquery-match-height.git#~0.7.0
bower matchHeight#~0.7.0                 resolve https://github.com/liabru/jquery-match-height.git#~0.7.0
bower patternfly-bootstrap-combobox#~1.1.7       not-cached https://github.com/patternfly/patternfly-bootstrap-combobox.git#~1.1.7
bower patternfly-bootstrap-combobox#~1.1.7          resolve https://github.com/patternfly/patternfly-bootstrap-combobox.git#~1.1.7
bower google-code-prettify#~1.0.5                not-cached https://github.com/tcollard/google-code-prettify.git#~1.0.5
bower google-code-prettify#~1.0.5                   resolve https://github.com/tcollard/google-code-prettify.git#~1.0.5
bower bootstrap-select#~1.10.0                   not-cached https://github.com/snapappointments/bootstrap-select.git#~1.10.0
bower bootstrap-select#~1.10.0                      resolve https://github.com/snapappointments/bootstrap-select.git#~1.10.0
bower eonasdan-bootstrap-datetimepicker#~4.17.37 not-cached https://github.com/Eonasdan/bootstrap-datetimepicker.git#~4.17.37
bower eonasdan-bootstrap-datetimepicker#~4.17.37    resolve https://github.com/Eonasdan/bootstrap-datetimepicker.git#~4.17.37
bower datatables-colreorder#~1.3.2               not-cached https://github.com/DataTables/ColReorder.git#~1.3.2
bower datatables-colreorder#~1.3.2                  resolve https://github.com/DataTables/ColReorder.git#~1.3.2
bower bootstrap-touchspin#~3.1.1                 not-cached https://github.com/istvan-ujjmeszaros/bootstrap-touchspin.git#~3.1.1
bower bootstrap-touchspin#~3.1.1                    resolve https://github.com/istvan-ujjmeszaros/bootstrap-touchspin.git#~3.1.1
bower d3#~3.5.17                                 not-cached https://github.com/mbostock-bower/d3-bower.git#~3.5.17
bower d3#~3.5.17                                    resolve https://github.com/mbostock-bower/d3-bower.git#~3.5.17
bower moment#~2.14.1                             not-cached https://github.com/moment/moment.git#~2.14.1
bower moment#~2.14.1                                resolve https://github.com/moment/moment.git#~2.14.1
bower angular#1.5.10                               resolved https://github.com/angular/bower-angular.git#1.5.10
bower get-style-property#1.x                        extract archive.tar.gz
bower eventie#~1.0.3                               download https://github.com/desandro/eventie/archive/v1.0.6.tar.gz
bower get-style-property#1.x                       resolved https://github.com/desandro/get-style-property.git#1.0.4
bower get-style-property#~1.0.4                     extract archive.tar.gz
bower get-style-property#~1.0.4                    resolved https://github.com/desandro/get-style-property.git#1.0.4
bower jquery#>=1.9.0                               resolved https://github.com/jquery/jquery-dist.git#3.3.1
bower angular-animate#1.3.0 - 1.5.*                download https://github.com/angular/bower-angular-animate/archive/v1.5.11.tar.gz
bower angular-bootstrap#0.14.x                     download https://github.com/angular-ui/bootstrap-bower/archive/0.14.3.tar.gz
bower eventEmitter#>=4.2 <5                        download https://github.com/Olical/EventEmitter/archive/v4.3.0.tar.gz
bower lodash#3.x                                   download https://github.com/lodash/lodash/archive/3.10.1.tar.gz
bower angular-sanitize#1.3.0 - 1.5.*               download https://github.com/angular/bower-angular-sanitize/archive/v1.5.11.tar.gz
bower c3#~0.4.11                                   download https://github.com/masayuki0812/c3/archive/v0.4.24.tar.gz
bower datatables-colvis#~1.1.2                     download https://github.com/DataTables/ColVis/archive/1.1.2.tar.gz
bower datatables#~1.10.12                          download https://github.com/DataTables/DataTables/archive/1.10.19.tar.gz
bower patternfly-bootstrap-treeview#~2.1.0       not-cached https://github.com/patternfly/patternfly-bootstrap-treeview.git#~2.1.0
bower patternfly-bootstrap-treeview#~2.1.0          resolve https://github.com/patternfly/patternfly-bootstrap-treeview.git#~2.1.0
bower doc-ready#~1.0.4                              extract archive.tar.gz
bower ev-emitter#~1.0.0                          not-cached https://github.com/metafizzy/ev-emitter.git#~1.0.0
bower ev-emitter#~1.0.0                             resolve https://github.com/metafizzy/ev-emitter.git#~1.0.0
bower bootstrap#>=2.3.2                             extract archive.tar.gz
bower doc-ready#~1.0.4                             resolved https://github.com/desandro/doc-ready.git#1.0.4
bower eventie#^1                                 not-cached https://github.com/desandro/eventie.git#^1
bower eventie#^1                                    resolve https://github.com/desandro/eventie.git#^1
bower eventie#^1                                   download https://github.com/desandro/eventie/archive/v1.0.6.tar.gz
bower bootstrap-datepicker#~1.6.4                  download https://github.com/eternicode/bootstrap-datepicker/archive/v1.6.4.tar.gz
bower font-awesome#~4.6.3                          download https://github.com/FortAwesome/Font-Awesome/archive/v4.6.3.tar.gz
bower eventie#~1.0.3                                extract archive.tar.gz
bower eventie#~1.0.3                               resolved https://github.com/desandro/eventie.git#1.0.6
bower bootstrap#>=2.3.2                            resolved https://github.com/twbs/bootstrap.git#4.2.1
bower patternfly-bootstrap-combobox#~1.1.7         download https://github.com/patternfly/patternfly-bootstrap-combobox/archive/1.1.7.tar.gz
bower matchHeight#~0.7.0                           download https://github.com/liabru/jquery-match-height/archive/0.7.2.tar.gz
bower google-code-prettify#~1.0.5                  download https://github.com/tcollard/google-code-prettify/archive/v1.0.5.tar.gz
bower jquery#~2.1.4                                 extract archive.tar.gz
bower eventEmitter#>=4.2 <5                         extract archive.tar.gz
bower eventEmitter#>=4.2 <5                    invalid-meta for:/tmp/32faccfb063ec5f39cbb248071d6f2c7/bower/60e2a10b5e51d8b8512f8623c13cf6f2-23-lMNjb4/bower.json
bower eventEmitter#>=4.2 <5                    invalid-meta The "name" is recommended to be lowercase, can contain digits, dots, dashes
bower eventEmitter#>=4.2 <5                        resolved https://github.com/Olical/EventEmitter.git#4.3.0
bower eventie#^1                                    extract archive.tar.gz
bower angular-animate#1.3.0 - 1.5.*                 extract archive.tar.gz
bower angular-sanitize#1.3.0 - 1.5.*                extract archive.tar.gz
bower eventie#^1                                   resolved https://github.com/desandro/eventie.git#1.0.6
bower angular-animate#1.3.0 - 1.5.*                resolved https://github.com/angular/bower-angular-animate.git#1.5.11
bower angular#1.5.11                             not-cached https://github.com/angular/bower-angular.git#1.5.11
bower angular#1.5.11                                resolve https://github.com/angular/bower-angular.git#1.5.11
bower angular#1.5.11                               download https://github.com/angular/bower-angular/archive/v1.5.11.tar.gz
bower bootstrap-select#~1.10.0                     download https://github.com/snapappointments/bootstrap-select/archive/v1.10.0.tar.gz
bower angular-sanitize#1.3.0 - 1.5.*               resolved https://github.com/angular/bower-angular-sanitize.git#1.5.11
bower datatables-colvis#~1.1.2                      extract archive.tar.gz
bower bootstrap#~3.3.7                              extract archive.tar.gz
bower eonasdan-bootstrap-datetimepicker#~4.17.37   download https://github.com/Eonasdan/bootstrap-datetimepicker/archive/4.17.47.tar.gz
bower datatables-colvis#~1.1.2                     resolved https://github.com/DataTables/ColVis.git#1.1.2
bower datatables#>=1.7.0                         not-cached https://github.com/DataTables/DataTables.git#>=1.7.0
bower datatables#>=1.7.0                            resolve https://github.com/DataTables/DataTables.git#>=1.7.0
bower datatables#>=1.7.0                           download https://github.com/DataTables/DataTables/archive/1.10.19.tar.gz
bower angular-bootstrap#0.14.x                      extract archive.tar.gz
bower angular-bootstrap#0.14.x                     resolved https://github.com/angular-ui/bootstrap-bower.git#0.14.3
bower datatables#~1.10.12                           extract archive.tar.gz
bower lodash#3.x                                    extract archive.tar.gz
bower jquery#~2.1.4                                resolved https://github.com/jquery/jquery-dist.git#2.1.4
bower bootstrap-touchspin#~3.1.1                   download https://github.com/istvan-ujjmeszaros/bootstrap-touchspin/archive/3.1.2.tar.gz
bower datatables-colreorder#~1.3.2                 download https://github.com/DataTables/ColReorder/archive/1.3.3.tar.gz
bower d3#~3.5.17                                   download https://github.com/mbostock-bower/d3-bower/archive/v3.5.17.tar.gz
bower font-awesome#~4.6.3                           extract archive.tar.gz
bower c3#~0.4.11                                    extract archive.tar.gz
bower patternfly-bootstrap-treeview#~2.1.0         download https://github.com/patternfly/patternfly-bootstrap-treeview/archive/v2.1.8.tar.gz
bower lodash#3.x                                   resolved https://github.com/lodash/lodash.git#3.10.1
bower patternfly-bootstrap-combobox#~1.1.7          extract archive.tar.gz
bower matchHeight#~0.7.0                            extract archive.tar.gz
bower datatables#~1.10.12                      invalid-meta for:/tmp/32faccfb063ec5f39cbb248071d6f2c7/bower/06669348abd69fdff10f03f7c7ff1571-23-IY0wx4/bower.json
bower datatables#~1.10.12                      invalid-meta The "main" field cannot contain font, image, audio, or video files
bower datatables#~1.10.12                      invalid-meta The "main" field cannot contain font, image, audio, or video files
bower datatables#~1.10.12                      invalid-meta The "main" field cannot contain font, image, audio, or video files
bower datatables#~1.10.12                      invalid-meta The "main" field cannot contain font, image, audio, or video files
bower datatables#~1.10.12                      invalid-meta The "main" field cannot contain font, image, audio, or video files
bower datatables#~1.10.12                      invalid-meta The "main" field has to contain only 1 file per filetype; found multiple .png files: ["media/images/sort_asc.png","media/images/sort_asc_disabled.png","media/images/sort_both.png","media/images/sort_desc.png","media/images/sort_desc_disabled.png"]
bower bootstrap-datepicker#~1.6.4                   extract archive.tar.gz
bower bootstrap-select#~1.10.0                      extract archive.tar.gz
bower patternfly-bootstrap-combobox#~1.1.7         resolved https://github.com/patternfly/patternfly-bootstrap-combobox.git#1.1.7
bower ev-emitter#~1.0.0                            download https://github.com/metafizzy/ev-emitter/archive/v1.0.3.tar.gz
bower moment#~2.14.1                               download https://github.com/moment/moment/archive/2.14.2.tar.gz
bower matchHeight#~0.7.0                       invalid-meta for:/tmp/32faccfb063ec5f39cbb248071d6f2c7/bower/e7492d2ba410ba5d41b5675732c04a29-23-LiGGTP/bower.json
bower matchHeight#~0.7.0                       invalid-meta The "name" is recommended to be lowercase, can contain digits, dots, dashes
bower matchHeight#~0.7.0                           resolved https://github.com/liabru/jquery-match-height.git#0.7.2
bower datatables#~1.10.12                          resolved https://github.com/DataTables/DataTables.git#1.10.19
bower datatables#>=1.7.0                            extract archive.tar.gz
bower bootstrap-touchspin#~3.1.1                    extract archive.tar.gz
bower datatables-colreorder#~1.3.2                  extract archive.tar.gz
bower angular#1.5.11                                extract archive.tar.gz
bower eonasdan-bootstrap-datetimepicker#~4.17.37    extract archive.tar.gz
bower bootstrap-touchspin#~3.1.1                   resolved https://github.com/istvan-ujjmeszaros/bootstrap-touchspin.git#3.1.2
bower d3#~3.5.17                                    extract archive.tar.gz
bower angular#1.5.11                               resolved https://github.com/angular/bower-angular.git#1.5.11
bower google-code-prettify#~1.0.5                   extract archive.tar.gz
bower d3#~3.5.17                                   resolved https://github.com/mbostock-bower/d3-bower.git#3.5.17
bower datatables-colreorder#~1.3.2                 resolved https://github.com/DataTables/ColReorder.git#1.3.3
bower ev-emitter#~1.0.0                             extract archive.tar.gz
bower patternfly-bootstrap-treeview#~2.1.0          extract archive.tar.gz
bower ev-emitter#~1.0.0                            resolved https://github.com/metafizzy/ev-emitter.git#1.0.3
bower eonasdan-bootstrap-datetimepicker#~4.17.37     invalid-meta for:/tmp/32faccfb063ec5f39cbb248071d6f2c7/bower/76bd398ee7174f6f95c3ab9b1ab3c397-23-iA30Wj/bower.json
bower eonasdan-bootstrap-datetimepicker#~4.17.37     invalid-meta The "main" field cannot contain minified files
bower eonasdan-bootstrap-datetimepicker#~4.17.37     invalid-meta The "main" field cannot contain minified files
bower c3#~0.4.11                                         resolved https://github.com/masayuki0812/c3.git#0.4.24
bower bootstrap#~3.3.7                                   resolved https://github.com/twbs/bootstrap.git#3.3.7
bower bootstrap-datepicker#~1.6.4                    invalid-meta for:/tmp/32faccfb063ec5f39cbb248071d6f2c7/bower/5302a8e90732aef5f2e93a0af3490637-23-uRVmyd/bower.json
bower bootstrap-datepicker#~1.6.4                    invalid-meta The "main" field cannot contain minified files
bower patternfly-bootstrap-treeview#~2.1.0               resolved https://github.com/patternfly/patternfly-bootstrap-treeview.git#2.1.8
bower bootstrap-datepicker#~1.6.4                        resolved https://github.com/eternicode/bootstrap-datepicker.git#1.6.4
bower moment#~2.14.1                                      extract archive.tar.gz
bower datatables#>=1.7.0                             invalid-meta for:/tmp/32faccfb063ec5f39cbb248071d6f2c7/bower/06669348abd69fdff10f03f7c7ff1571-23-JL8zHk/bower.json
bower datatables#>=1.7.0                             invalid-meta The "main" field cannot contain font, image, audio, or video files
bower datatables#>=1.7.0                             invalid-meta The "main" field cannot contain font, image, audio, or video files
bower datatables#>=1.7.0                             invalid-meta The "main" field cannot contain font, image, audio, or video files
bower datatables#>=1.7.0                             invalid-meta The "main" field cannot contain font, image, audio, or video files
bower datatables#>=1.7.0                             invalid-meta The "main" field cannot contain font, image, audio, or video files
bower datatables#>=1.7.0                             invalid-meta The "main" field has to contain only 1 file per filetype; found multiple .png files: ["media/images/sort_asc.png","media/images/sort_asc_disabled.png","media/images/sort_both.png","media/images/sort_desc.png","media/images/sort_desc_disabled.png"]
bower google-code-prettify#~1.0.5                    invalid-meta for:/tmp/32faccfb063ec5f39cbb248071d6f2c7/bower/8b46b49e999a9774c040fd46f81a68a5-23-Up8tKe/bower.json
bower google-code-prettify#~1.0.5                    invalid-meta The "main" field cannot contain minified files
bower google-code-prettify#~1.0.5                    invalid-meta The "main" field cannot contain minified files
bower eonasdan-bootstrap-datetimepicker#~4.17.37         resolved https://github.com/Eonasdan/bootstrap-datetimepicker.git#4.17.47
bower moment#>=2.10.5                                  not-cached https://github.com/moment/moment.git#>=2.10.5
bower moment#>=2.10.5                                     resolve https://github.com/moment/moment.git#>=2.10.5
bower moment#>=2.10.5                                    download https://github.com/moment/moment/archive/2.24.0.tar.gz
bower datatables#>=1.7.0                                 resolved https://github.com/DataTables/DataTables.git#1.10.19
bower google-code-prettify#~1.0.5                        resolved https://github.com/tcollard/google-code-prettify.git#1.0.5
bower bootstrap-select#~1.10.0                           resolved https://github.com/snapappointments/bootstrap-select.git#1.10.0
bower font-awesome#~4.6.3                                resolved https://github.com/FortAwesome/Font-Awesome.git#4.6.3
bower moment#~2.14.1                                     resolved https://github.com/moment/moment.git#2.14.2
bower moment#>=2.10.5                                     extract archive.tar.gz
bower moment#>=2.10.5                                    resolved https://github.com/moment/moment.git#2.24.0

Please note that,
angular-route#1.5.10 depends on angular#1.5.10 which resolved to angular#1.5.10
angular-patternfly#3.17.0 depends on angular#1.3.0 - 1.5.* which resolved to angular#1.5.10
angular-bootstrap#0.14.3 depends on angular#>=1.3.0 which resolved to angular#1.5.10
angular-animate#1.5.11, angular-sanitize#1.5.11 depends on angular#1.5.11 which resolved to angular#1.5.11
Resort to using angular#1.5.10 which resolved to angular#1.5.10
Code incompatibilities may occur.

bower                                                   ECONFLICT Unable to find suitable version for bootstrap

npm ERR! Linux 3.10.0-862.el7.x86_64
npm ERR! argv "/opt/rh/rh-nodejs6/root/usr/bin/node" "/opt/rh/rh-nodejs6/root/usr/bin/npm" "install"
npm ERR! node v6.11.3
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE
npm ERR! web@1.0.0 postinstall: `bower --config.registry.search=${BOWER_MIRROR} install`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the web@1.0.0 postinstall script 'bower --config.registry.search=${BOWER_MIRROR} install'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the web package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     bower --config.registry.search=${BOWER_MIRROR} install
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs web
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls web
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /opt/app-root/src/npm-debug.log
error: build error: non-zero (13) exit code from docker-registry.default.svc:5000/openshift/nodejs@sha256:9a9b1e60650ef4060957ef8fc3a89f63a996d794c7be55c5f233671cace8945f



oc describe is/nodejs -n openshift
Name:			nodejs
Namespace:		openshift
Created:		24 hours ago
Labels:			<none>
Annotations:		openshift.io/display-name=Node.js
			openshift.io/image.dockerRepositoryCheck=2019-01-22T10:40:46Z
Docker Pull Spec:	docker-registry.default.svc:5000/openshift/nodejs
Image Lookup:		local=false
Unique Images:		5
Tags:			7

0.10
  tagged from registry.redhat.io/openshift3/nodejs-010-rhel7:latest
    prefer registry pullthrough when referencing this tag

  DEPRECATED: Build and run Node.js 0.10 applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-nodejs-container/blob/master/0.10/README.md.
  Tags: hidden, nodejs
  Supports: nodejs:0.10, nodejs:0.1, nodejs
  Example Repo: https://github.com/openshift/nodejs-ex.git

  * registry.redhat.io/openshift3/nodejs-010-rhel7@sha256:cd7a62da7dbd15883b7df9a8a1ffb18499bfec5e856661f96c5126ae51281b83
      24 hours ago

10 (latest)
  tagged from registry.redhat.io/rhoar-nodejs/nodejs-10
    prefer registry pullthrough when referencing this tag

  Build and run Node.js 10 applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/bucharest-gold/centos7-s2i-nodejs.
  Tags: builder, nodejs
  Example Repo: https://github.com/openshift/nodejs-ex.git

  * registry.redhat.io/rhoar-nodejs/nodejs-10@sha256:ea0c9c67d10b5204d51666d3d34078627f4ef24683ab45e0e6768b0b51b4853d
      24 hours ago

4
  tagged from registry.redhat.io/rhscl/nodejs-4-rhel7:latest
    prefer registry pullthrough when referencing this tag

  Build and run Node.js 4 applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-nodejs-container/blob/master/4/README.md.
  Tags: hidden, builder, nodejs
  Supports: nodejs:4, nodejs
  Example Repo: https://github.com/openshift/nodejs-ex.git

  ! error: Import failed (InternalError): Internal error occurred: Get https://registry.redhat.io/v2/rhscl/nodejs-4-rhel7/manifests/latest: received unexpected HTTP status: 503 Service Unavailable
      24 hours ago

6
  tagged from registry.redhat.io/rhscl/nodejs-6-rhel7:latest
    prefer registry pullthrough when referencing this tag

  Build and run Node.js 6 applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-nodejs-container.
  Tags: builder, nodejs
  Supports: nodejs:6, nodejs
  Example Repo: https://github.com/openshift/nodejs-ex.git

  * registry.redhat.io/rhscl/nodejs-6-rhel7@sha256:9a9b1e60650ef4060957ef8fc3a89f63a996d794c7be55c5f233671cace8945f
      24 hours ago

8
  tagged from registry.redhat.io/rhscl/nodejs-8-rhel7:latest
    prefer registry pullthrough when referencing this tag

  Build and run Node.js 8 applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-nodejs-container.
  Tags: builder, nodejs
  Example Repo: https://github.com/openshift/nodejs-ex.git

  * registry.redhat.io/rhscl/nodejs-8-rhel7@sha256:1a42ca9592ea96f1d9b6042cab05191e68ba7759e98b25334bd6af04cd028b35
      24 hours ago

8-RHOAR
  tagged from registry.redhat.io/rhoar-nodejs/nodejs-8
    prefer registry pullthrough when referencing this tag

  Build and run Node.js 8 applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/bucharest-gold/centos7-s2i-nodejs.
  Tags: builder, nodejs
  Example Repo: https://github.com/openshift/nodejs-ex.git

  * registry.redhat.io/rhoar-nodejs/nodejs-8@sha256:89f3a61476fd5317b557ce67f617fa310c259130285decdd6e1590797a00b4ba
      24 hours ago

--- a/web-nodejs/bower.json
+++ b/web-nodejs/bower.json
@@ -17,6 +17,7 @@
     },
     "resolutions": {
         "angular": "1.5.10",
+        "bootstrap": "3.3.7",
         "jquery": "3.2.1"
     }
 }

